src-electron/generator/matter/app/zap-templates/templates/app/helper.js

```diff
@@ -717,57 +765,28 @@ async function zapTypeToClusterObjectType(type, isDecodable, options) {
         return 'uint' + s[1] + '_t';
       }
 
-      const bitmapObj = await zclQuery.selectBitmapByName(
+      const clusterCount = await getClusterCountForType(
         this.global.db,
         pkgId,
-        type
-      );
-      const ns = nsValueToNamespace(
-        options.hash.ns,
-        bitmapObj.bitmapClusterCount
+        type,
+        'Bitmap',
```

Comment:
Minor cleanup: Switch to `db-enum.zclType.bitmap` ? same for enums and structs

Reply:
The problem is the casing @brdandu .  In the function I need the strings "bitmap" and "Bitmap".

db-enum.zclType.bitmap is "bitmap", so to get "Bitmap" you need to do some complicated "uppercase the first letter, append the rest" thing.  To get "bitmap" from "Bitmap" you just lowercase the whole thing.  So passing in Bitmap here makes the callee code a lot simpler.

But I can make this change if desired.

Reply:
Should a new set of enums be created?

Reply:
I don't think it's worth creating a separate set of enums.  If you think enum should be used here, I can just do the uppercase first letter thing.

Reply:
Per Slack discussion, keeping this as-is for now.